tests/phpunit/helpers/class-wc-subscriptions-change-payment-gateway.php

```
@@ -11,7 +11,7 @@ class WC_Subscriptions_Change_Payment_Gateway {
 	 * @param WC_Subscription $subscription Subscription to update.
 	 * @param string          $gateway_id   Gateway ID.
 	 */
-	public static function update_payment_method( $subscription, $gateway_id ) {}
+	public static function update_payment_method( $subscription, $gateway_id, $token = null ) {}
```

PHPMD (2.15.0)

[error] The method update_payment_method is not named in camelCase. (undefined)

(CamelCaseMethodName)